firmware/tunerstudio/rusefi.input

```diff
@@ -1553,27 +1553,36 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		field = "Idle Solenoid Frequency",   			idle_solenoidFrequency, !useStepperIdle
 	
 	dialog = idleStepper, "Stepper"
-		field = "Idle Stepper Step",					idle_stepperStepPin, useStepperIdle
-		field = "Idle Stepper Dir",						idle_stepperDirectionPin, useStepperIdle
-		field = "Idle Stepper Enable",					stepperEnablePin, useStepperIdle
-		field = "Idle Stepper Enable mode",				stepperEnablePinMode, useStepperIdle
+		field = "Idle Stepper Step Pin",				idle_stepperStepPin, useStepperIdle
+		field = "Idle Stepper Dir Pin",					idle_stepperDirectionPin, useStepperIdle
+		field = "Idle Stepper Enable Pin",				stepperEnablePin, useStepperIdle
+		field = "Idle Stepper Enable Pin Mode",			stepperEnablePinMode, useStepperIdle
 		field = "Stepper reaction time",				idleStepperReactionTime, useStepperIdle
 		field = "Stepper total steps",					idleStepperTotalSteps, useStepperIdle
 		field = "Stepper parking extra steps, %",		stepperParkingExtraSteps, useStepperIdle
 		field = "Force parking every restart",			stepperForceParkingEveryRestart, useStepperIdle
 		
 	dialog = idleHwType, "Idle Valve Hardware",	border
-		field = "use stepper",							useStepperIdle
```

Comment:
Moved to 'idlehw' panel to place this setting above both stepper & solenoid panels.